miso-dto/src/main/java/uk/ac/bbsrc/tgac/miso/dto/Dtos.java

```diff
@@ -1493,13 +1501,21 @@ public static LibraryDto asDto(@Nonnull Library from, boolean includeBoxPosition
     if (requisition != null) {
       setId(dto::setRequisitionId, requisition);
       setString(dto::setRequisitionAlias, requisition.getAlias());
+      dto.setRequisitionStopped(requisition.isStopped());
+      dto.setRequisitionPaused(requisition.getPauses().stream().anyMatch(
+          pause -> (pause.getEndDate() == null || pause.getEndDate().isAfter(LocalDate.now()))
+      ));
```

Comment:
Remove these lines. Already being done below outside of the if/else that handles direct vs. effective requisitions